### ISSUE TITLE
Design System

### ISSUE THREAD

**Issue by abbyhipp** [hackforla/HomeUniteUs#269]
### Overview
We need to have a Design System to be able to make our designs universal

### Action Items
- [x] Move Figma board into drafts
- [x] Organize Figma board: decide who needs to have edit/view access
- [x] Begin component audit and create shareable tickets
 
- [x] Change typography in design system
### Resources/Instructions

**Comment by juliaa423** [comment]
Going along with the 2nd point, I organized names of some of the pages with the following naming convention: 

**Type: Page Description (Name of person who worked on it)**
Ex. Wireframes: Desktop Guest Portal (Amira)

**Comment by juliaa423** [comment]
Updates 2/2: 4 groups of prototypes need to be audited. (2 mobile flows from calendaring, 1 dashboard/1 application from current HUU Figma board). Audit is being done in 🔦 Design System Audit - IN PROGRESS page on Figma. 
1. Use Frame “AUDIT - Basics: Guest/Host Hi Fid Prototype 2.0” as basis for upcoming typography + color audits. 
2. Judy is assigned to the 2nd mobile flow (AUDIT - Basics: Case Worker Hi Fid Prototype 2.0). Alyssa and Kshitij to be assigned remaining items after they onboard. 
3. Next steps: I will start the component audit (ex. Buttons, icons) for the Calendaring: Guest/Host Hi Fid Prototype 2.0 frame and update here once complete.

**Comment by ariamacabre** [comment]
@juliaa423 I wanted to point out something I noticed with the color comments/audit in the style guide. There is some confusion, which is understandable since the Primary Accent color wasn't fully updated after Beata redid it in the original section. The Primary Accent color #55B1EB is not accessible for color contrast, so Beata updated it in the box that says "Color Blender" below to #249BE5 -- and this is why the colors in Calendaring don't match. I wanted to point this out because I realized you might not have known this 😊
